extensions/gitlab/src/mrmenu.tsx

```diff
@@ -62,7 +63,7 @@ export default function MenuCommand(): JSX.Element {
           >
             {mrsAssigned?.map((m) => (
               <MenuBarItem
-                icon={"mropen.png"}
+                icon={{ source: GitLabIcons.merge_request, tintColor: Color.PrimaryText }}
```

Comment:
This was the reason for this PR, the `Color.PrimaryText` didn't appear to have any effect on the PNG icon in the Menu Bar. 

While I was here, also changed to use the GitLabIcons enum for all references for the "mropen.png".